Src/AutoFixture/CustomizeAttribute.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Reflection;
+
+namespace Ploeh.AutoFixture
+{
+    /// <summary>
+    /// Base class for customizing parameters in methods.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Parameter, AllowMultiple = true)]
+    public abstract class CustomizeAttribute : Attribute
```

Comment:
It looks I got stuck creating the Greedy Passive Attribute and [GreedyAttributeRelay](https://github.com/sergeyshushlyapin/AutoFixture/blob/passive-attributes/Src/AutoFixture.xUnit.net2/GreedyAttributeRelay.cs#L29). In the Relay I need to resolve the specimen using the original `request` (which contains the specimen type) and `GreedyConstructorQuery`:

```csharp
var methodInvoker = new MethodInvoker(new GreedyConstructorQuery());
return context.Resolve(request /*, methodInvoker*/);
```

Just cannot figure out how to create such kind of specimen. Any help is highly appreciated! Thanks.